src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big-js';
+import Symbols from '../symbols';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [operand1, operand2] = [numberOne, numberTwo].map(x => Big(x));
+  switch (operation) {
+    case Symbols.ADDITION:
+      return operand1.plus(operand2);
+    case Symbols.DIVISION:
+      return operand1.div(operand2);
```

Comment:
dont forget to manage division m=by ZERO either in this milestone or the next one (it will generate an error)😃

Reply:
Thanks, I will look out for it.